cmd/mattermost/commands/user.go

```diff
@@ -45,6 +45,14 @@ var UserCreateCmd = &cobra.Command{
 	RunE:    userCreateCmdF,
 }
 
+var UserModifyCmd = &cobra.Command{
+	Use:     "modify [emails, usernames, userIds] --bot",
```

Comment:
@jasonblais I'm not very happy with using the "modify" command to covert a user in a bot. For me is an explicit action that should be made independently. I would expect that user modify allow me to modify the users data, like email, username, first_name, nickname or whatever, but not change it entirely to a different thing, in this case a Bot.

I would suggest to add the command `user convert-to-bot [usernames,emails,userids]` or maybe `user covert [usernames, emails, userids] --bot`.

Reply:
@jespino A bot would still exist in the Users table, and I could see this command extending to modify other properties of a user or a bot in future, like its username or display name.

I was basing it off of converting a channel from public to private, and vice versa https://docs.mattermost.com/administration/command-line-tools.html#mattermost-channel-modify

Reply:
Yes, but that is a very concrete property of the channel and in this case is not so clear. Anyway I'm ok with that if we define how to do the inverse action, and we add some tickets (can be help wanted tickets) for add other modification features here.

The point is, if we have a command for modify an account and we have the `--bot` flag we can do 2 things, allowing things like `--bot true` (convert regular user to bot), `--bot false` (convert bot to regular user) and no passing the flag, which makes no action in this case (in the future, when you pass other flags you want to handle the flags that are passed and ignore to rest). Another option would be use `--bot` and `--no-bot` or `--regular-user` or `--user`.

Reply:
After thinking about it some more, I can see potential problems of scaling this to modifying specific attributes of the user. 

So instead, the proposal would be to do the following as you suggested:

```
// convert a user to bot
mattermost user convert [emails, usernames, userIds] --bot

// convert a bot to a user with email/password login
mattermost user convert botusername —-user --email new.email@email.com --password password

// modify a user’s username
Mattermost user modify [emails, usernames, userIds] —username <new-username>

// modify a user’s email address
Mattermost user modify [emails, usernames, userIds] —email <new-email-address>
```

Of course, only the first of these (converting user to bot) is in the scope of this PR.

Is this in alignment with what you had in mind @jespino?

Reply:
Yes, that is exactly what I had in mind I think is more concrete about the actions that you are trying to do, and don't generate false expectations about the possibility of modify the users.

Reply:
@liusy182 To summarize the discussion on this thread, the proposal is to update the command to

```
mattermost user convert [emails, usernames, userIds] --bot
```

This would be similar for converint a bot to a user with email/password login

```
mattermost user convert botusername —-user --email new.email@email.com --password password
```

Reply:
sounds good. agree that convert is better here